src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -108,13 +108,11 @@ private boolean isTestingStudentData(
         return isTestingData;
     }
 
-    private StatsBundle generateStatsPerInstitute(
-            List<CourseStudent> allStudents, List<Instructor> allInstructors, List<Account> allAccounts) {
-        HashMap<String, HashMap<Integer, HashSet<String>>> institutes =
-                new HashMap<String, HashMap<Integer, HashSet<String>>>();
-
-        HashSet<String> allInstructorEmailSet = new HashSet<String>();
-        HashSet<String> allStudentEmailSet = new HashSet<String>();
+    private StatsBundle generateStatsPerInstitute(List<CourseStudent> allStudents, List<Instructor> allInstructors,
+                                                  List<Account> allAccounts) {
```

Comment:
This is an unrelated change; also, the previous style of indentation is preferred.